lib/parseBinaryGltf.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+var fs = require('fs');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var defaultValue = Cesium.defaultValue;
+var StringDecoder = require('string_decoder').StringDecoder;
+module.exports = parseBinaryGltf;
+
+function parseBinaryGltf(data) {
+    var sizeOfUint32 = Uint32Array.BYTES_PER_ELEMENT;
+    var byteOffset = 0;
+
+    if (data.slice(byteOffset, sizeOfUint32).toString() !== 'glTF') {

```

Comment:
I think `slice` looks directly into the memory of the buffer subset - https://nodejs.org/api/buffer.html#buffer_buf_slice_start_end. The documentation says that the Node Buffer is an implementation of the Uint8Array Typed Array.